### PR TITLE
added snackbar

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

File: css/components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.hide{
+  display: none;
+}
+
+.snackbar{
+  width: 20rem;
+  background-color: var(--sec-col);
+  color: var(--prim-col);
+  padding: 0.5rem 1rem;
+  border-radius: 0.5rem;
```
Comment (malaykhakhar) [inline css/components/snackbar.css:10]: It is advisable to use px when rem value is less than 1.

File: js/snackbar.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const snackTrigger= document.querySelectorAll('.snackbar-trigger');
+const snackBar= document.querySelector('.snackbar');
+
+snackTrigger.forEach(trigger=>{
+  console.log("clicken snackbar")
```
Comment (malaykhakhar) [inline js/snackbar.js:5]: Logging to the console is not needed.

**Review by Megha-Joshi — APPROVED** [review]
Looks good to me. Can be merged

**Review by malaykhakhar — CHANGES_REQUESTED** [review]
Rest of the code looks good.

File: js/snackbar.js (hunk 2 above)
Comment (sanjitsarkar) [inline js/snackbar.js:5]: Remove console logging in production

File: templates/snackbar.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
```
Comment (sanjitsarkar) [inline templates/snackbar.html:5]: Title tag should not be empty

File: css/components/snackbar.css (hunk 1 above)
Comment (sanjitsarkar) [reply css/components/snackbar.css:10]: Yes use px for smaller than 1rem

**Review by sanjitsarkar — APPROVED** [review]

**Pull request merged**